### PR TITLE
Shattered halls

### PR DESCRIPTION
**Pull request by miraco**
### Proof

- None

### Issues

- None

### How2Test

- None

### Todo / Checklist

- [X] None

### CODE REVIEW

**Comment by AnonXS** [comment]
please make different branches for the different things you want to fix, polluting your master branch with this isnt gonna work out in the long run, make a development branch where you can merge your fixing branches into and the development branche then can be rebased by master branch which is syncronized with cmangos-tbc

**Comment by al3xc1985** [comment]
Nice! This would fix some existing bugs

File: sql/updates/mangos/s2411_01_mangos_shattered_halls.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-- Shattered Hand Legionnaire now gets handled via Core Script
+UPDATE creature_template SET AIName="",  ScriptName = "mob_shattered_hand_legionnair" WHERE Entry="16700";
```
Comment (AnonXS) [inline sql/updates/mangos/s2411_01_mangos_shattered_halls.sql:2]: https://github.com/cmangos/mangos-tbc/blob/master/sql/scriptdev2/scriptdev2.sql#L673

File: sql/updates/mangos/s2411_01_mangos_shattered_halls.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+DELETE FROM creature_ai_scripts WHERE creature_id = "16700";
+
+
+-- Shattered Hand Legionnaire Call for Reinforcements texts
+INSERT INTO `script_texts` (`entry`, `content_default`, `content_loc1`, `content_loc2`, `content_loc3`, `content_loc4`, `content_loc5`, `content_loc6`, `content_loc7`, `content_loc8`, `sound`, `type`, `language`, `emote`, `broadcast_text_id`, `comment`) VALUES
```
Comment (AnonXS) [inline sql/updates/mangos/s2411_01_mangos_shattered_halls.sql:7]: https://github.com/cmangos/mangos-tbc/blob/master/sql/scriptdev2/scriptdev2.sql#L3754

File: sql/updates/mangos/s2411_01_mangos_shattered_halls.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
+-- Shattered Hand Legionnaire now gets handled via Core Script
+UPDATE creature_template SET AIName="",  ScriptName = "mob_shattered_hand_legionnair" WHERE Entry="16700";
+DELETE FROM creature_ai_scripts WHERE creature_id = "16700";
```
Comment (AnonXS) [inline sql/updates/mangos/s2411_01_mangos_shattered_halls.sql:3]: Is automatically deleted when it's removed from ACID.sql file

File: sql/scriptdev2/scriptdev2.sql [hunk 4]
```diff
@@ -659,6 +659,7 @@ UPDATE instance_template SET ScriptName='instance_ramparts' WHERE map=543;
 
 /* SHATTERED HALLS */
 /* Nethekurse and his spawned shadowfissure */
+UPDATE creature_template SET AIName="",  ScriptName = "mob_shattered_hand_legionnair" WHERE Entry="16700";
```
Comment (AnonXS) [inline sql/scriptdev2/scriptdev2.sql:662]:
```
-- ============================================================
-- Remove All Creature Template Entries That Conflict With ACID
-- ============================================================
UPDATE creature_template SET AIName= '' WHERE AIName= 'EventAI';
UPDATE creature_template SET ScriptName= '' WHERE ScriptName= 'generic_creature';


-- ================================================
-- Define ACID Full Release Script Creature Entries
-- ================================================
UPDATE creature_template INNER JOIN creature_ai_scripts ON creature_template.entry = creature_ai_scripts.creature_id SET AIName='EventAI';
```

https://raw.githubusercontent.com/cmangos/tbc-db/master/ACID/acid_tbc.sql

no need to set AIName

**Comment by miraco** [comment]
Im gonna split this in multiplte prs.

**Comment by al3xc1985** [comment]
Have u done prs for all of rhem? I saw only 2 in the pull req

**Comment by miraco** [comment]
No.
Have to much to do irl and im playing Shadowlands atm. Im having free time over chrismas/new year maybe i can finish it then.

**Comment by al3xc1985** [comment]
@miraco anyway to have some time for PRs?

**Comment by miraco** [comment]
I already uploaded the Ai changes, gonna upload my shattered halls script when im finished with the last group before the first boss. (Maybe in the next to days or next weekend)

**Comment by al3xc1985** [comment]
love ya..ty

**Pull request closed without merging**